sdk/webpubsub/azure-messaging-webpubsub/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Release History
 
+## 1.3.0 (2024-06-28)
+
+- Added a `clientEndpointType` option to `GenerateClientTokenOptions` to specify the type of client endpoint
+  when generating token. This option can be used to generate token and client connection URL for a specific client endpoint type, such as `Default` or `MQTT`.
+
+#### Dependency Updates
+
+- Updated `api-version` to `2024-01-01`.
+
+
 ## 1.3.0-beta.1 (Unreleased)
 
 - Migrated serialization to `azure-json` which offers implementation agnostic serialization, providing support for
```

Comment:
Please merge the section with 1.3.0-beta.1
(1.3.0-beta.1 is not released, so the changes there also applies to 1.3.0)

Also please add a section of below (the exact lib version depends on your release date -- if you follow the July release, you will need to wait a few days for new azure-core)

```
### Other Changes
#### Dependency Updates
- Upgraded `azure-core` from `1.47.0` to version `1.48.0`.
- Upgraded `azure-core-http-netty` from `1.14.1` to version `1.14.2`.
```

Reply:
So for the Dependency Updates, we just add a temporary section like what you suggested, then we'll change it to the correct lib version when we actually release right?

Reply:
What's your scheduled release date?

If you don't have a date yet, I'd suggest you put e.g. `1.3.0 (Unreleased)`, without the "Dependency Updates" section.
You will use the Prepare-Release script to update the CHANGELOG upon your release (when you can add the "Dependency Updates" section with exact version).